src/makeCalculator.js

```diff
@@ -38,6 +38,47 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {};
+
+  calculator.result = 0;
+
+  calculator.add = (num) => {
+    calculator.result += num;
+
+    return calculator.result;
+  };
```

Comment:
You should add methods and static fields while initialising object
```suggestion
  const calculator = {
    result = 0;
    add (num) {
      //...
    }
    //... 
  };

```